src/interfaces/store-interfaces/state-interfaces/business-interface.ts

```diff
@@ -38,5 +38,6 @@ export interface BusinessIF {
   taxId?: string // aka Business Number // may be undefined
   state: EntityState
   stateFiling?: string
+  startDate: ApiDateTimeUtc | string
```

Comment:
I believe `ApiDateTimeUtc` is an alias for string, so you shouldn't need the union type.